### PR TITLE
UUID update.

### PR DESCRIPTION
**Pull request by Njs41**
- Store ignored and muted players in MySQL using UUIDs instead of user names.

- Adds back in schema query for shadow mutes that was previously removed to remove it properly

### CODE REVIEW

File: src/no/runsafe/nchat/channel/ChannelManager.java [hunk 1]
```diff
@@ -49,11 +49,14 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
 		if (cmp == 0)
 			return null;
 
-		if (ignoreHandler.playerIsIgnoring(player1Name, player2Name))
-			return null;
+		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
+		{
+			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2))
```
Comment (mortenn) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: could playerIsIgnoring be made to check both ways? if not, could we merge these two ifs?

Comment (Njs41) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: It's used only one way in a few other places, so I'll merge the if statements

File: src/no/runsafe/nchat/channel/ChannelManager.java [hunk 2]
```diff
@@ -49,14 +49,11 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
...
 
-		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
-		{
-			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2))
-				return null;
-
-			if (ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1))
-				return null;
-		}
+		if ((player1 instanceof IPlayer && player2 instanceof IPlayer)
```
Comment (mortenn) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: Maybe not merge all three, because this is verging on the cryptic :p

Comment (mortenn) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: Also, when are either player not an IPlayer, and what happens below this if then?

Comment (Njs41) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: Then one of them is likely the console, and I'm not exactly sure what the logic behind getting the channel name is doing

Comment (mortenn) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: The channel name for PMs is %<someone>-<else> to make sure nobody ever creates a normal channel with that name.

File: src/no/runsafe/nchat/channel/ChannelManager.java [hunk 3]
```diff
@@ -49,11 +49,11 @@ public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor
...
-		if ((player1 instanceof IPlayer && player2 instanceof IPlayer)
-			&& (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2)
-			|| ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1))
-		)
-			return null;
+		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
+			if (ignoreHandler.playerIsIgnoring((IPlayer) player1, (IPlayer) player2)
+				|| ignoreHandler.playerIsIgnoring((IPlayer) player2, (IPlayer) player1)
+			)
+				return null;
```
Comment (mortenn) [inline src/no/runsafe/nchat/channel/ChannelManager.java]: This still feels very messy.. I think we should just add a `eitherPlayerIsIgnoring` and use that.. saves on redundant casting too.

**Pull request merged**